config/default.yml

```diff
@@ -589,6 +589,12 @@ PreCommit:
     include:
       - '**/*.html'
 
+  LineEndings:
+    description: 'Check line endings'
+    enabled: false
+    eol: 'lf'
+    # eol: 'crlf'
```

Comment:
I don't remember exactly why I switched to `lf`/`crlf`, but probably influenced by `git ls-files` as mentioned below. Yes going with literal characters would even be better. I'll add this to the next iteration.